src/js/form-builder.js

```diff
@@ -963,6 +964,10 @@ function FormBuilder(opts, element, $) {
       if (attribute === 'label' && !opts.disableHTMLLabels) {
         inputConfig.contenteditable = true
         attributefield += m('div', attrVal, inputConfig).outerHTML
+      } else if (typeVal.includes('textarea') && attribute === 'value') {
+        inputConfig.contenteditable = true
+        inputConfig.value = attrVal
+        attributefield += `<textarea ${attrString(inputConfig)}>${inputConfig.value}</textarea>`
```

Comment:
```suggestion
      } else if (values.type === 'textarea' && attribute === 'value') {
        attributefield += m('textarea', attrVal, inputConfig).outerHTML
```

textarea doesn't need the contenteditable set and we can use the markup helper rather than manually constructing the HTML element

Reply:
Your suggested change looks good to me.  I was a little unhappy with the backtick manual html.  Was just following the lines below this code which sets up a single input line 975, but using the m function is the way forward.